spec/requests/api/v1/partners_requests_spec.rb

```diff
@@ -26,4 +26,52 @@
       expect(result["agency"]).to be_present
     end
   end
+
+  describe 'POST /api/v1/partners' do
+    context 'when the body is empty' do
+      it 'responds with an error status code' do
+        post api_v1_partners_path({})
+
+        expect(response).to have_http_status(:bad_request)
+      end
+    end
+
+    context 'when given valid parameters' do
+      it 'responds with an :ok status' do
+        valid_partner_creation_request
+
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'creates a new `Partner` record' do
+        expect { valid_partner_creation_request }
+          .to change { Partner.count }
+          .from(0)
+          .to(1)
+      end
+
+      it 'responds with the `id` and `email` of the new record' do
+        valid_partner_creation_request(email: 'new.partner@example.com')
+
+        result = JSON.parse(response.body)
+
+        expect(result).to have_key('id')
+        expect(result).to have_key('email')
+        expect(result['email']).to eq('new.partner@example.com')
+      end
+    end
+  end
+
+  def valid_partner_creation_request(
```

Comment:
I'm not super happy with how this ended up looking stylistically, but I wasn't sure what sort of style limits/preferences the project has. Please let me know if you have any suggestions on how to improve!